content/install/installation_overview.md

```diff
@@ -52,6 +52,8 @@ services:
       - DRONE_SECRET=${DRONE_SECRET}
 ```
 
+By adding more `drone-agents`, you can scale to multiple builds in parallel. 
```

Comment:
"You can add more agents to increase the number of parallel builds."

Reply:
Thanks, sorry... Not a very good English speaker/writer...